mlir/include/mlir/Query/Matcher/VariantValue.h

```diff
@@ -92,8 +93,15 @@ class VariantValue {
   const VariantMatcher &getMatcher() const;
   void setMatcher(const VariantMatcher &matcher);
 
+  // Unsigned value functions.
+  bool isUnsigned() const;
+  unsigned getUnsigned() const;
+  void setUnsigned(unsigned Unsigned);
```

Comment:
MLIR's naming convention is camelBack (https://mlir.llvm.org/getting_started/DeveloperGuide/#style-guide).